fixtures/large/rspec_mocks_proxy_expected.rb

```diff
@@ -350,7 +352,7 @@ def message_received(message, *args, &block)
       def any_instance_class_recorder_observing_method?(klass, method_name)
         only_return_existing = true
         recorder = ::RSpec::Mocks.space.any_instance_recorder_for(klass, only_return_existing)
-        return true if recorder && recorder.already_observing?(method_name)
```

Comment:
it is correct for standard to make this change, it's not a change rubyfmt can make